cmake/SetupCharm.cmake

```diff
@@ -1,7 +1,16 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-find_package(Charm 6.10.2 EXACT REQUIRED)
+# Apple Silicon (arm64) Macs require charm 7.0.0 to run
+if(APPLE)
```

Comment:
does
```
if (APPLE AND "${CMAKE_HOST_SYSTEM_PROCESSOR}" STREQUAL "arm64")
  find_package(Charm 7.0.0 EXACT REQUIRED)
else()
  find_package(Charm 6.10.2 EXACT REQUIRED)
endif()
```
work?

Reply:
Yes, this seems to work...I did not know AND was available for make if...thanks for the tip! Fixed here and throughout